thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx

```diff
@@ -79,79 +82,75 @@ export const TrendingAnomalies: FunctionComponent = () => {
     }, [currentChartZoom]);
 
     useEffect(() => {
-        getAnomalies({ startTime: generateDateRangeMonthsFromNow(6)[0] }).then(
-            (anomalies) => {
-                if (!anomalies) {
-                    return;
+        getAnomalies({
+            startTime: startTime || generateDateRangeMonthsFromNow(6)[0],
```

Comment:
Let's force the parent to always pass a startTime.

Reply:
Updated